samples/README.md

```diff
@@ -2,38 +2,49 @@
    samplefwlink:  https://aka.ms/WinIoTSamples
 --->
 
-# EFLOW Internet of Things (IoT) Samples
+# Azure EFLOW Samples
 
-This repo contains samples that demonstrate usage patterns for interoperatibility between Microsoft's Windows 10 and the EFLOW VM.  These code samples were created with templates available in Visual Studio and are designed, but not limited to, run on devices using EFLOW. For more information, review [EFLOW docs](https://docs.microsoft.com/en-us/windows/iot/iot-enterprise/azure-iot-edge-for-linux-on-windows).
```

Comment:
I think this was true before (because guides at first were generic to Win10 IoT), but now are specific to EFLOW, so we should  remove that "but not limited to"